Makefile

```diff
@@ -57,7 +57,7 @@ ifneq (,$(RM_DIRS))
 endif
 
 scaladoc:
-	$(SBT) $(SBT_FLAGS) doc
+	$(SBT) $(SBT_FLAGS) unidoc
```

Comment:
What's the effect of this change? Is there a difference between "sbt doc" and "sbt unidoc"?

Reply:
`sbt unidoc` generates scaladoc for all subprojects. If you look at the scaladoc generated by `sbt doc`, you'll find it's missing the definitions from coremacros and more importantly, chiselFrontend.

Reply:
Good point; would be nice to put this into the commit message.

Reply:
I've updated the PR body to mention this.